napari/_qt/qt_main_window.py

```diff
@@ -35,6 +35,7 @@
     QToolTip,
     QWidget,
 )
+from superqt import ensure_main_thread
```

Comment:
```suggestion
from superqt import ensure_object_thread
```

as we talk about Qt objects here, I prefer to use `ensure_object_thread` instead of `ensure_main_thread`. I know that currently, this will produce the same result but will look more proper to me.